marblerun/features/auto-injection.md

```diff
@@ -39,9 +39,14 @@ metadata:
 
 We use this label to map Kubernetes Pods to MarbleRun Marbles.
 When you deploy your application, MarbleRun will read out this label's value, `voting-svc` in the example above.
-It will check the Marbles section of your manifest for an entry with the same name.
-If such entry is present in the manifest, the Pod is provided with the particular configuration for this Marble.
-If no such entry exists or a valid `marblerun/marbletype` label is missing, the Pod's creation is rejected.
+It will then inject environment variables and SGX resources into the containers of the Pod.
+If the `marblerun/marbletype` label is missing, the Pod's injection is skipped.
+
+## The Marblecontainer label
+
+By default MarbleRun will inject environment variables and resource requests into all containers of the Pod.
+You can use the `marblerun/marblecontainer=<ContainerName>` label to limit injection to the specified container.
```

Comment:
```suggestion
You can use the `marblerun/marblecontainer=<ContainerName>` label to limit injection to the specified container.
```
Do we want to limit this to a single container? Or can this be a list?

Reply:
I don't think Kubernetes supports defining labels as lists.
Even if we could, we would then inject the environment for one Marble into multiple containers, which doesn't really make sense imo.

Reply:
> Even if we could, we would then inject the environment for one Marble into multiple containers, which doesn't really make sense imo.

Mhhm, good point.